### PR TITLE
Did everything!

### PR DESCRIPTION
**Pull request by danedavid**

### CODE REVIEW

File: human.txt [hunk 1]
```diff
@@ -31,8 +32,8 @@ jsstrn
 justyn-clark
 kenhrmt
 kylecho
-Leo-Mu
 leo424y
+Leo-Mu
```
Comment (callumacrae) [inline human.txt]: Why have you moved this? 😱

Comment (danedavid) [reply human.txt]: I thing it was moved when I ran the `sort` utility

Comment (aaossa) [reply human.txt]: Can you fix it please? Once you commit those changes we'll be able to merge your PR. Thanks @danedavid

Comment (SubhrajyotiSen) [reply human.txt]: Actually, he sorted the entire list. So that should be fine since we get a fully sorted list in the end.

Comment (danedavid) [reply human.txt]: So.. it is ?

Comment (callumacrae) [reply human.txt]: image

`"-" < "4"` 🤔

**Comment by danedavid** [comment]
Now ok ?

File: human.txt (hunk 1 above)
Comment (SubhrajyotiSen) [reply human.txt]: Oops! I was wrong. Thanks for the sort clarification

**Comment by SubhrajyotiSen** [comment]
Yes. It's fine now

**Pull request merged**